src/Controller/BaseImportControllerTrait.php

```diff
@@ -118,7 +126,7 @@ protected function doImportSave(Request $request, TranslatorInterface $translato
 
         $this->setErrorAsFlash($form->getErrors(true));
```

Comment:
` $this->setErrorAsFlash($form->getErrors());`, because we don't need form errors as global errors.